.changeset/spotty-walls-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': patch
+---
+
+Fixed a bug in the `KubernetesProxy endpoint` where requests to clusters configured with `client-side auth providers` would always fail with a 500 status.
```

Comment:
maybe I'm missing something, but why the `inline code block` syntax here?
```suggestion
Fixed a bug in the Kubernetes proxy endpoint where requests to clusters configured with client-side auth providers would always fail with a 500 status.
```

Reply:
I tend to get a lot of vale errors in changeset messages when mentioning words like kubernetes, so i inline the words to state these are words that make sense to the code-base but might not necessarily make sense in a sentence. Just something ive done as a chore when submitting PR's.

Reply:
the word "kubernetes" should be allowed:

https://github.com/backstage/backstage/blob/08bd36717d79dee994e5d7fe1690a889c41b10ea/.github/vale/Vocab/Backstage/accept.txt#L184

The thing about putting words in code blocks is that then they don't get spell-checked at all! This is OK for actual code symbols like class names but "client-side auth provider" is supposed to be a phrase for humans